### ISSUE TITLE
Option to choose only one crop mode.

### ISSUE THREAD

**Issue by gaurava-mishra** [Andrew0000/ImagePickerPlus#3]
Hello, 
This is not an issue, but a possibile feature that could be widely used.

I am working an application that need final image to be scaled in 512x512 ( i.e. square with custome pixel value).

I could able to use this library though, but app will crash if user select some other crop option rather than square like 16:9, 4:3 5:4 etc.

I have working experience with uCrop earlier, and I knew that uCrop provides options to retrict the user to crop in any particular ratio. 
I think, you guys doing good job to combine picker and cropper in a single frame. 
Could you also consider this feature request?

**Comment by Andrew0000** [comment]
Hello @gaurava-mishra,

If I understand you correctly, 512px is the upper limit and it's ok if an image is smaller than 512px. I just added the ability to restrict the possible aspect ratio: `strictAspectRatio` in `ImageTransformation`. So if my guess is correct, you will be able to restrict the request like this:
```
PickRequest(
    source = ...,
    transformation = ImageTransformation(
        maxSidePx = 512,
        strictAspectRatio = AspectRatioIPP("1:1", 1f, 1f),
    ),
)
```
Please update the dependency version to 1.0.3 and try it.

**Comment by gaurava-mishra** [comment]
Thanks Alot.
I will check it out.

**Comment by gaurava-mishra** [comment]
I have checked it and it is working fine. One more this we could notice,
Let say if an user wants to load image having its requirement to be minimum 512 px (in 1:1) cropper.

Could you please add minSidePx  in this library?

**Comment by Andrew0000** [comment]
Let me break your question into possible stages:
- Limit the ability to choose an image with the wrong size in the system's picker. I'm not sure if the system picker provides this ability.
- Limit the ability to crop an image on the cropping stage. Is that what you need? What if a user already picked an image smaller than required?
- Upscale the final image if it's smaller than required. I believe it's not a good option.

So basically at this point I don't see the reliable solution. As far as I know we cannot prevent user from picking small images from the system. I may be wrong.
